src/main/resources/META-INF/mods.toml

```diff
@@ -0,0 +1,32 @@
+modLoader="javafml"
+loaderVersion="${loaderVersionRange}"
+license="GNU LGPLv3"
```

Comment:
This can be updated to say MIT

Reply:
Changed in commit https://github.com/The-Fireplace-Minecraft-Mods/Simply-Caterpillar/pull/4/commits/77f6b7c222da62089d04745baa56b7c1047e442e